packages/rebber/src/types/tableRow.js

```diff
@@ -9,8 +9,18 @@ const defaultMacro = (ctx, node) => {
   return `${line} \\\\ \\hline\n`
 }
 
+const defaultFirstLineRowFont = `\\rowfont[c]{\\bfseries}`
+const defaultOtherLineRowFont = `\\rowfont[l]{}`
+
 /* Stringify a tableRow `node`. */
-function tableRow (ctx, node) {
+function tableRow (ctx, node, index) {
   const macro = ctx.tableRow || defaultMacro
-  return macro(ctx, node)
+  const firstLineRowFont = ctx.firstLineRowFont || defaultFirstLineRowFont
```

Comment:
Is the first line actually a header?

Reply:
good question, in fact this was the old behaviour and i kept it here because I still don't know how to resolve the true header